doc/source/programs/gdal_raster_create.rst

```diff
@@ -0,0 +1,138 @@
+.. _gdal_raster_create_subcommand:
+
+================================================================================
+"gdal raster create" sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Create a new raster dataset.
+
+.. Index:: gdal raster create
+
+Synopsis
+--------
+
+.. program-output:: gdal raster create --help-doc
+
+Description
+-----------
+
+:program:`gdal raster create` can be used to initialize a new raster file,
+from its dimensions, band count and set various parameters, such as CRS,
+geotransform, nodata value, metadata.
+
+The new file can also be initialized from a model input file with the optional
+:option:`--input` option, copying its properties but not its pixel values.
```

Comment:
I see there is an alias `-like`. Maybe I like it more.

Reply:
changed to --like